src/main/java/Parser.java

```diff
@@ -0,0 +1,62 @@
+//package duke;
+/**
+ * translate command -> name and time
+ */
+public class Parser {
```

Comment:
I really like how your Parser class separates different functions in a very intuitive manner to readers. However, perhaps you could provide a more detailed naming to the functions? This is so that the functionality of your functions can be more easily inferred by its name.